sdk/core/azure-core/src/main/java/com/azure/core/util/logging/ClientLogger.java

```diff
@@ -173,6 +172,17 @@ private RuntimeException logException(RuntimeException runtimeException, LogLeve
         return runtimeException;
     }
 
+    /**
+     * Determines if the environment and logger support logging at the given log level.
+     *
+     * @param logLevel The {@link LogLevel} being validated as supported.
+     * @return Flag indicating if the environment and logger support logging at the given log level.
+     */
+    public boolean canLogAtLevel(LogLevel logLevel) {
+        LogLevel environmentLoggingLevel = getEnvironmentLoggingLevel();
+        return canLogAtLevel(logLevel, environmentLoggingLevel);
```

Comment:
No need to create a local variable here - just call `getEnvironmentLoggingLevel()` directly in the second line.

Reply:
Done